libs/accounts/email-renderer/src/partials/automatedEmailRecoveryKey/index.ts

```diff
@@ -0,0 +1,15 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+export type TemplateData = {
+  device?: {
+    uaBrowser: string;
+    uaOS: string;
+    uaOSVersion: string;
```

Comment:
Are these always present? In `emails/partials/userDevice`, it looks like only `uaOS` is basically guaranteed. It's possible the template safeguards against too much (or too little?) though.

edit: whoops, meant to start a review